knowledge-base/docker/metadata-action/action-security.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Docker Metadata action'
+github-token:
+  action-input:
+    input: github-token
+    is-default: true
+  permissions:
+    contents: read
+    contents-reason: to query repo information
```

Comment:
Same here. We should start adding link to reason next to the `-reason`. Makes it easy for me to click on the link and understand/ confirm it.